Development to Distribution

TOUnail opened the pull request:
- Added Block to display Galleries
- Added missing button variants
- Change Event Card design from Dentistry to Figma/Sketch design

krcardoza committed:
setting up editor side gallery block

TOUnail committed:
remove namespace for gallery block, register block through metadata

krcardoza committed:
add styling for preview page

krcardoza committed:
responsive css for preview

krcardoza committed:
remove build folder

krcardoza committed:
add alt text to preview page images, condition when no featured image

TOUnail committed:
fix conflicts

TOUnail committed:
merge with development branch

krcardoza committed:
Merged in gallery-block (pull request #12)

Gallery Block

Approved-by: Alex Tou

TOUnail committed:
compile build

TOUnail committed:
Merged distribution into development

TOUnail committed:
add more button variants, fix hero height

TOUnail committed:
replace event card design to match comp lib prototype

TOUnail committed:
change event card style to fit prototype

TOUnail committed:
Merged feature/add-missing-variants into development

TOUnail committed:
compile build

krcardoza commented:
secondary dark button can't be seen on the editor side unless you hover over it. I understand that its for dark backgrounds but not entirely sure how this button would be used in our templates. Also the play button isn't working on my end. The image might have been hosted locally.

TOUnail committed:
fix all style for button, add end time for events, fix event card design to reflect design prototype

TOUnail commented:
> secondary dark button can't be seen on the editor side unless you hover over it. I understand that its for dark backgrounds but not entirely sure how this button would be used in our templates. Also the play button isn't working on my end. The image might have been hosted locally.

Button styles in the editor has been fixed. In the editor, I've changed the Secondary Dark variant to be a slightly darker shade of white so it can be seen on light backgrounds. I've also fixed the Play button.

krcardoza commented:
![Screen Shot 2022-10-04 at 2 09 29 PM](https://user-images.githubusercontent.com/112980789/193929672-41bdaa3d-d691-4103-921b-a384a7680658.png)
getting this error whenever i refresh the editor page.

TOUnail committed:
fix empty content for hero banner, fix button validation

TOUnail committed:
add defaults to attributes

TOUnail committed:
change attribute

TOUnail committed:
fix button styles in gutenberg

TOUnail committed:
fix event end time

TOUnail committed:
fix end time for upcoming events

TOUnail committed:
fix single page event time

TOUnail committed:
fix event end time, add photo credit field to hero block

TOUnail committed:
fix photo credit preview

TOUnail committed:
fix photo credit position

TOUnail committed:
revert hero photo credit
